package.json

```diff
@@ -93,6 +93,9 @@
     "hooks": {
       "pre-commit": [
         "lint-staged"
+      ],
+      "pre-push": [
```

Comment:
I prefer no pre-push hooks, as I want the push to always succeed. Then we run tests on CI.